libs/@guardian/libs/src/consent-management-platform/getConsentFor.ts

```diff
@@ -8,10 +8,7 @@ export const getConsentFor: GetConsentFor = (
 ): boolean => {
 	const sourcepointIds = VendorIDs[vendor];
```

Comment:
If you wanted to stop checking whether this was defined you could do a nullish coalescing to an empty array:

```ts
const sourcepointIds = VendorIDs[vendor] ?? [];
```